osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -96,6 +96,7 @@ protected override void LoadComplete()
         {
             updateTooltipText(Current.Value);
             base.LoadComplete();
+            CurrentNumber.ValueChanged += updateTooltipText;
```

Comment:
Use `CurrentNumber.BindValueChanged(updateTooltipText, true);` (replaces the manual call just above.